frontend/src/FirstRun/AuthenticationRequiredModalContent.js

```diff
@@ -95,27 +97,38 @@ function AuthenticationRequiredModalContent(props) {
                 />
               </FormGroup>
 
-              <FormGroup>
-                <FormLabel>{translate('Username')}</FormLabel>
-
-                <FormInputGroup
-                  type={inputTypes.TEXT}
-                  name="username"
-                  onChange={onInputChange}
-                  {...username}
-                />
-              </FormGroup>
+              {
+                authenticationEnabled ?
```

Comment:
I don't think there is a lot of value in hiding this until the user makes a selection since they're going to have to fill it out regardless, whereas it might be hidden in settings because of how `External` works.